lua/lspsaga/diagnostic/init.lua

```diff
@@ -278,10 +278,10 @@ function diag:render_diagnostic_window(entry, option)
 
   local virt = {}
   if entry.source then
-    virt[#virt + 1] = { entry.source, 'Comment' }
+    virt[#virt + 1] = { entry.source, 'SagaVirtLine' }
```

Comment:
this is not virtual line.  so use `SagaVirtLine` a bit inappropriate

Reply:
thank you for review